bin/quality

```diff
@@ -2,7 +2,12 @@
 
 set -e
 
+# bin/rubocop --rails --format json --config config/rubocop.yml
```

Comment:
commented until guys decide if they want to use these tools and the manner of using